tests/integration/lmic_test_builder.py

```diff
@@ -267,7 +267,7 @@ def log_metrics(self, sequence):
             file = open("llm/metrics.log", "r")
             metrics = re.sub("'", r'"', file.readline())
             command = f'aws cloudwatch put-metric-data --namespace "LMIC_performance_{sequence["engine"]}" ' \
-                      f'--region "us-east-1" --metric-data "{metrics}"'
+                      f'--region "us-east-1" --metric-data \'{metrics}\''
```

Comment:
should `\'{metrics}\'` be `\"{metrics}\"` ?

Reply:
The double quotes weren't parsing properly, so we moved it over to single quotes and confirmed that it was working properly for posting the metrics json in cloudwatch. Just needed the escape character since we were in a f'' string.